docs/development/issue_labeling.md

```diff
@@ -3,10 +3,21 @@
 We try to keep issues well-classified by labeling them appropriately.
 Only members/collaborators with the required rights can apply labels.
 
-## Severity labels
+## Basic knowledge about Renovate
+
+You should know about platforms, package managers, datasources and versioning to label issues effectively.
+
+- To learn about platforms, read the [Renovate docs on Platforms](https://docs.renovatebot.com/modules/platform/).
+- To learn about managers, read the [Renovate docs on Managers](https://docs.renovatebot.com/modules/manager/).
+- To learn about datasources, read the [Renovate docs on Datasources](https://docs.renovatebot.com/modules/datasource/).
+- To learn more about versioning, read the [Renovate docs on Versioning](https://docs.renovatebot.com/modules/versioning/).
+
+## Label categories
+
+### Severity
 
 <details>
-    <summary>Severity labels</summary>
+    <summary>Severity</summary>
 
     breaking
     major
```

Comment:
I removed `major` now and I'm not sure that `breaking` then needs its own category. It can maybe be related to feature/fix/refactor ?

Reply:
I've removed the `major` label, and moved the breaking label to the Type of issue section.
I'm not sure what `breaking` denotes though, see the new file for questions. :smile: